README.md

```diff
@@ -253,6 +253,13 @@ interface to get information if the usage of space and index names in requests
 is supported.
 * `Schema` structure no longer implements `SchemaResolver` interface.
 
+#### Client tools changes
+
+* `Op` struct made private.
+* Remove `OpSplice` struct.
+* `Operations` struct made an instance of `msgpack.CustomEncoder` interface.
```

Comment:
You don't need to notion this, it doesn't break anything.

Reply:
Updated.